psi4/src/psi4/dfmp2/corr_grad.cc

```diff
@@ -656,12 +656,13 @@ void DFCorrGrad::build_AB_x_terms() {
         int oQ = auxiliary_->shell(Q).function_index();
 
         int ncart = cP * cQ;
-        const double* Px = buffer + 0 * ncart;
-        const double* Py = buffer + 1 * ncart;
-        const double* Pz = buffer + 2 * ncart;
-        const double* Qx = buffer + 3 * ncart;
-        const double* Qy = buffer + 4 * ncart;
-        const double* Qz = buffer + 5 * ncart;
+        bool libint1 = Process::environment.options.get_str("INTEGRAL_PACKAGE") != "LIBINT2";
```

Comment:
Should this rather be `== "LIBINT"`?

Reply:
Or switch it around so that the variable is `libint2`.

Reply:
Also, since the indexing is the same, you could define `const double *bufp = libint2 ? Jint[thread]->buffers() : buffer` and then the 6 lines of assignment wouldn't need the conditionals.

Reply:
These are all going to be removed when Libint1 goes away.  I just needed to hack something in temporarily to get both libint1 and libint2 to compile simultaneously, for debugging and testing.